lib/game_stats.rb

```diff
@@ -23,21 +23,21 @@ def percentage_home_wins
     home_wins = @games_collection.games.count do |game|
       game.home_goals > game.away_goals
     end
-    (home_wins.to_f / @games_collection.games.count * 100).round(3)
+    (home_wins.to_f / @games_collection.games.count).round(2)
```

Comment:
Thank you for getting the * 100 out of our methods. We're getting the percentages we want now.